.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on: [pull_request]
```

Comment:
Nitpicking: you don't need to wrap it in an array

Reply:
👍